api/README.md

```diff
@@ -66,6 +66,84 @@ Tests are placed in `e2e/tests/` directory, in the `e2e` package. The entrypoint
 1. As the `environment_name` is supposed to be set in the test data, these tests are currently only able to run in the `dev` environment. For running tests in `staging` / `production` environments, some changes will be required.
 2. Test runs are isolated by using different MLP projects and/or TEST_ID (see the [Local](#local) section below). This will prevent conflicts in the cluster resources, if tests are run simultaneously. However, the BQ outcome table is also shared and thus, tear down will not remove it.
 
+### Running End-to-end tests locally
+
+This is a work in progress and there changes required on this process as this is extremely clunky. A rough way of running the e2e tests are as follows:
+
+First, spin up k8s and other services required for Turing:
+
+```bash
+#!/bin/bash
+
+pushd ../infra/docker-compose/dev/
+{
+    docker-compose up -d
+}
+popd
+```
+
+Here we have to wait for all the pods to be ready.
+
+```bash
+watch KUBECONFIG=/tmp/kubeconfig kubectl get pod -A
```

Comment:
`brew install watch` on Mac for this first?

Reply:
lol i thought watch was a standard command, standard tool when you install linux, the person should be able to Bing this.

Reply:
The first I've heard "Bing this" :P 

Cool.